packages/payload/uploads.d.ts

```diff
@@ -1,2 +1,2 @@
-export { getFileByPath } from './dist/uploads/getFileByPath.js';
-//# sourceMappingURL=uploads.d.ts.map
\ No newline at end of file
+export { getFileByPath } from './dist/uploads/getFileByPath.js'
```

Comment:
Honestly I would like to just see these reverted since they are unrelated to PR.